tools/validators/instance_validator/tests/telemetry_validator_test.py

```diff
@@ -48,7 +52,9 @@ def ack(self):
 
 with open(os.path.join(_TELEMETRY_PATH,
                        'telemetry_good.json')) as file:
-  _MESSAGE_GOOD = FakeMessage(_MESSAGE_ATTRIBUTES_1, file.read())
+  file_contents = file.read()
+  _MESSAGE_GOOD = FakeMessage(_MESSAGE_ATTRIBUTES_1, file_contents)
+  _MESSAGE_GOOD_2 = FakeMessage(_MESSAGE_ATTRIBUTES_3, file_contents)
```

Comment:
should these go to a a def setUp ?

Reply:
Not sure. It looks like a setUp method would run before each test, so that would lead to a lot of unnecessary file I/O instead of just reading the test data once.

I see that there's a setUpClass method (https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUpClass), but it's unclear to me if this is strictly advantageous over just defining these test constants in file scope as we're currently doing.